src/file_operations.rs

```diff
@@ -140,6 +155,42 @@ mod tests {
     }
 
     #[test]
+    fn can_link_file() {
+        let test_root = set_up("link");
+        let mut file_set = FileOperations::rooted_at(&test_root);
+
+        let mut f = fs::File::create(test_root.join("file_a")).unwrap();
+        
+        file_set.link("file_a", "file_b");
+        let results = file_set.commit();
+        assert_eq!(results.len(), 1);
+        assert!(results[0].is_ok());
+
+        let attr = fs::symlink_metadata(test_root.join("file_b")).unwrap();
+        assert!(attr.file_type().is_symlink());
+
+        clean_up(&test_root);
+    }
+
+    #[test]
+    fn can_unlink_file() {
+        let test_root = set_up("unlink");
+        let mut file_set = FileOperations::rooted_at(&test_root);
+
+        // Create symbolic link to remove
+        let mut f = fs::File::create(test_root.join("file_a")).unwrap();
+        file_set.link("file_a", "file_b");
```

Comment:
This doesn't need to be a symlink since it's a generic remove operation, so we can probably ditch the indirection and just attempt to remove `file_a`.